README.md

```diff
@@ -10,19 +10,18 @@ React Router keeps your UI in sync with the URL. It has a simple API with powerf
 ### Docs & Help
 
 - [Tutorial – do this first!](https://github.com/reactjs/react-router-tutorial)
-- [Guides and API docs](https://github.com/reactjs/react-router/tree/master/docs)
+- [Guides and API docs](/docs)
```

Comment:
These links don't need to be fully qualified. npm resolves these correctly as well, e.g. for the `/SPONSORS.md` link below.